pxl_scripts/px/redis_data/redis_data.pxl

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' Redis Data Tracer
+
+This script traces all Redis data on the cluster.
```

Comment:
nit but the script itself doesn't trace the events. this script displays the most recent events